index.test-d.ts

```diff
@@ -0,0 +1,20 @@
+import { expectType, expectAssignable } from 'tsd'
+import * as closeWithGrace from '.'
+import { Options, CloseWithGraceCallback, Signals } from '.'
+
+expectAssignable<CloseWithGraceCallback>(async (param: { err: Error, manual: boolean, signal: Signals }) => { })
```

Comment:
We need to test also partial parameters being passed since the run function pass only one parameter per call.